app/controllers/password_resets_controller.rb

```diff
@@ -37,6 +38,12 @@ def user_params
     params.require(:user).permit(:password, :password_confirmation)
   end
 
+  def find_token
+    @user = User.find_by(email: params[:email])
```

Comment:
Thanks @Enriikke! I'll adjust once the Encryptor class is setup. Just to be clear: you mean for me to remove `find_token` all together and, in `find_user`, `find_by` the `hashed_token`/`reset_digest`, yeah? Once I get rid of the `User`'s `reset_token_salt` attribute and replace it with the secret key, that should be no problem. Right now I need the `@user` to get that user's specific salt but we can change that based on your previous comment.